platform/src/error_code.rs

```diff
@@ -226,3 +226,10 @@ pub enum ErrorCode {
     N01016 =  1016, N01017 =  1017, N01018 =  1018, N01019 =  1019, N01020 =  1020,
     N01021 =  1021, N01022 =  1022, N01023 =  1023,
 }
+
+impl ErrorCode {
+    pub fn into_u16(self) -> u16 {
+        // SAFETY: `ErrorCode` is #[repr(u16)], so this cast is safe.
+        unsafe { core::mem::transmute(self) }
+    }
+}
```

Comment:
I think you can convert an `ErrorCode` into a `u16` using `as`.

Reply:
Whoops! Good catch haha